### ISSUE TITLE
Saxon error not propagating to UI

### ISSUE THREAD

**Issue by marcolarosa** [CoEDL/nyingarn-workspace#58]
@sophlew uploaded a transcription document to `SLNSW_FL814347` in the workspace which resulted in the following error in the task runner node:

```
today at 4:28:17 PM2022-05-31T06:28:17.189Z INFO: Running 'process-tei' task for 'SLNSW_FL814347' in /tmp/05aae298-5d99-4760-8afe-d55854e5c0bd
today at 4:28:17 PMERROR: unpaginated document. No pages could be extracted from the file because none found in the document.
today at 4:28:17 PM    at template Q{http://www.w3.org/1999/XSL/Transform}initial-template in process-tei-to-page-files.xsl#13
today at 4:28:17 PMTransformation failure: Error Q{http://www.w3.org/2005/xqt-errors}XTMM9000 at process-tei-to-page-files.xsl#40
today at 4:28:17 PM    Terminated with ERROR: unpaginated document. No pages could be extracted from the file because none found in the document.
today at 4:28:17 PMError
today at 4:28:17 PM    at new K (/srv/node_modules/saxon-js/SaxonJS2N.js:3887:513)
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4423:130
today at 4:28:17 PM    at Array.L (/srv/node_modules/saxon-js/SaxonJS2N.js:4135:641)
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4154:511
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4136:33
today at 4:28:17 PM    at Array.forEach (<anonymous>)
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4136:6
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4164:69
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4164:69
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4164:69
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4164:69
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4164:69
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4164:69
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4136:33
today at 4:28:17 PM    at Array.forEach (<anonymous>)
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4136:6
today at 4:28:17 PM    at Object.push (/srv/node_modules/saxon-js/SaxonJS2N.js:4167:370)
today at 4:28:17 PM    at e (/srv/node_modules/saxon-js/SaxonJS2N.js:4757:104)
today at 4:28:17 PM    at /srv/node_modules/saxon-js/SaxonJS2N.js:4784:306 {
today at 4:28:17 PM  message: 'Terminated with ERROR: unpaginated document. No pages could be extracted from the file because none found in the document.',
today at 4:28:17 PM  name: 'XError',
today at 4:28:17 PM  code: 'Q{http://www.w3.org/2005/xqt-errors}XTMM9000',
today at 4:28:17 PM  xsltLineNr: '40',
today at 4:28:17 PM  xsltModule: 'process-tei-to-page-files.xsl'
today at 4:28:17 PM}
```

But for some reason this error didn't show in the UI.

**Comment by sophlew** [comment]
I worked it out =) The page numbering inside the document was not correct. SLNSW_FL814347 is not the item for this test. It needed to be SLNSW_FL814 and then SLNSW_FL814-347 inside the document. 
I also removed the table from Stephen's document for the test. The SLNSW numbering system and how they have since changed it will be a challenge. Stephen has word transcriptions (with old numbering) and manuscript images (with new numbering) ARGH!!! 
![Screenshot (176)](https://user-images.githubusercontent.com/91518014/171111140-8e3f81f2-cf17-4f59-a870-b9b4f5886385.png)
